scripts/database-setup.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+
+# environment variables
+source ./scripts/variables.sh
+
+echo
+echo "Creating database table"
+echo
+
+echo "==== member table ===="
+psql \
+-c \
+  "CREATE TABLE IF NOT EXISTS member (
+    member_id VARCHAR(80) PRIMARY KEY,
+    first_name VARCHAR(30),
+    last_name VARCHAR(30),
+    email_address VARCHAR(320) UNIQUE NOT NULL,
+    password VARCHAR(72) NOT NULL,
+    profile_picture TEXT,
+    is_verified BOOLEAN DEFAULT false,
+    is_owner BOOLEAN DEFAULT false,
+    is_moderator BOOLEAN DEFAULT false,
+    is_blocked BOOLEAN DEFAULT false,
+    created_at timestamp DEFAULT current_timestamp
+  );" \
+-d $PG_DATABASE \
+-h $PG_HOST \
+-p $PG_PORT \
+-U $PG_USER
```

Comment:
@CodingCreate101 any way to avoid writing these 4 lines again and again.

Reply:
It could be used as a variable.
Ex:
`CONFIG=-d $PG_DATABASE -h $PG_HOST -p $PG_PORT -U $PG_USER`

and use `$CONFIG` wherever we want.

Not sure if the syntax is accurate but something like that could be possible. You might have already tried this though.

Reply:
Hmm...

I tried to convert this part into a function and then use it, but `psql` command was converting the name of the function as database name.

I can try with the code you have mentioned, like converting the code into the variable itself.